lib/spack/spack/util/module_cmd.py

```diff
@@ -1,4 +1,4 @@
-# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
```

Comment:
Why revert the copyright (2021 -> 2020)?

Reply:
Ah, sorry, oversight! Copied the file from our fork directly…

Reply:
Np.